charts/sophora-server/values.yaml

```diff
@@ -328,3 +328,7 @@ resources:
     memory: 16Gi
   limits:
     memory: 16Gi
+
+podDisruptionBudget:
+  enabled: false
+  minAvailable: 2
```

Comment:
```
podDisruptionBudget:
  ## @param enabled Whether the pod disruption budget resource should be deployed
  enabled: false
  ##
  ## @param podDisruptionBudget.minAvailable Mininimum number of pods that must still be available after the eviction
  ##
  minAvailable: 1
  ## @param podDisruptionBudget.maxUnavailable Max number of pods that can be unavailable after the eviction
  ##
  maxUnavailable: ""
```

(based on https://github.com/bitnami/charts/blob/main/bitnami/cassandra/values.yaml#L404) 

To "test" this feature, add some arbitrary pdb config to `test-values.yaml` :)